roles/splunk_cluster_master/tasks/initialize_multisite_master.yml

```diff
@@ -0,0 +1,22 @@
+---
+- include_tasks: ../../../roles/splunk_common/tasks/wait_for_splunk_instance.yml
+  vars:
+    splunk_instance_address: "{{ splunk.multisite_master }}"
+
+- name: Convert Extrenal Cluster Master Name into Internal URI
+  set_fact:
+      multisite_master_uri: "https://{{ splunk.multisite_master }}:{{ splunk.multisite_master_port }}"
```

Comment:
Might want to use {{ cert_prefix }} here instead of "https" because of my recent changes

Reply:
Thanks, I'll do that